### DISCUSSION TITLE
Add details/support for XFS MS-DOS Client!

### DISCUSSION THREAD

**Discussion by cml37** [retronas/discussions#56] [Ideas]
Hello!

So, another option for MS-DOS file sharing is the XFS client!

I've written a few procedures on how to use it: https://github.com/cml37/dos-utils/tree/master/network/xfs

It would basically involved enabling NFSv2 if not already enabled, and then perhaps adding an article to the Wiki on how to configure it.

Glad to help with this too.

**Comment by sairuk** [comment]
Converted to discussion.

**Reply by sairuk** [reply]
We would happily take/transfer a wiki article from the community on this. I did have this setup years ago in MAME as a test but its been far too long i'd have to go and try setting it up up again.

**Comment by danmons** [comment]
We have NFSv2 already:
* https://github.com/danmons/retronas/wiki/NFS

Looks like "XFS" is an MS-DOS NFSv2 implementation for DOS packet drivers:
* http://omolini.steptail.com/mirror/winsite/winsite-mar96-3/info/pc/win3/winsock/xfs191.zip/index.html

We should have everything ready to go on the RetroNAS side I think.

**Reply by cml37** [reply]
Correct!  I think that adding a wiki article offering some configuration advice is pretty much all that is needed.
